### ISSUE TITLE
P.Setting is lack of one setting making playlist changed in order

### ISSUE THREAD

**Issue by swallowchen1359** [WishCanTaiwan/VocabuLazy#108]
Currently playlist changed either random or play the same list always.
Users can only change playlist in order by scroll manually.

Should implement one more setting for the circumstance.
To @allencheng07 
I will have a new pull request tomorrow completing the layout and preparing the callback function for you to implement.

**Comment by swallowchen1359** [comment]
@tomliou please help to check after the issue finished.

**Comment by swallowchen1359** [comment]
@allencheng07 
After #113, layout is implemented, please help to complete the callback function.
